pkg/oci/skopeo.go

```diff
@@ -1,10 +1,14 @@
 package oci
 
 import (
+	"fmt"
 	"os"
 	"os/exec"
+	"os/user"
 )
 
+var registries = []string{"docker.io"}
```

Comment:
why this and the for loop? 🤔

Reply:
there are multiple registri skopeo supports